cloud/pkg/synccontroller/synccontroller.go

```diff
@@ -222,6 +240,58 @@ func (sctl *SyncController) manageObjectSync(syncs []*v1alpha1.ObjectSync) {
 		}
 	}
 }
+func (sctl *SyncController) DeleteOutdatedObjectSync() {
+	for {
+		select {
+		case <-beehiveContext.Done():
+			klog.Warning("Stop synccontroller DeleteOutdatedObjectSync loop")
+			return
+		case e := <-sctl.nodeManager.Events():
+			node, ok := e.Object.(*v1.Node)
```

Comment:
Is it possible to reuse the nodeinformer in synccontroller to get the node delete event? @ls889 @GsssC

Reply:
@fisherxu  I try to reuse it today, but failed. Because nodeinformer does'n expose lw. we can't access the event from informer.